phpunit.xml.dist

```diff
@@ -21,11 +21,11 @@
         <extension class="Symfony\Component\Panther\ServerExtension" />
     </extensions>
 
-    <filter>
-        <whitelist>
+    <coverage>
```

Comment:
Do you want to do the same PR on dev-kit ?

Reply:
Yep, that was the idea.